lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -94,6 +113,7 @@ _InsertionLocation _insertionLocationForPackageImport(
   final AstNode relativeNode;
   final bool insertAfter;
   final bool inOwnSection;
+  bool hasPackageImports = true;
```

Comment:
This name is not clear, as below you're now setting it to false if there are any relative imports at all. Should maybe be hasRelativeImports?

Reply:
This `bool` is for enabling true condition where we want imports with the package and if it is not `package` then we are setting it to false.

Reply:
OK. Maybe it should be hasOnlyPackageImports? Or hasAllPackageImports?